### PR TITLE
sig_figs, troubleshooting for CmdStan guide

### PR DESCRIPTION
**Pull request by mitzimorris**
#### Submission Checklist

- [x] Builds locally 
- [x] Declare copyright holder and open-source license: see below

#### Summary

Updated CmdStan guide for addition of `sig_figs` output arg in 2.25.
Also added troubleshooting section to installation docs

Also added code font for arg `theta` for Bernoulli distress in Functions Reference because deemed not worth separate PR.

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):   Columbia University

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

### CODE REVIEW

**Comment by mitzimorris** [comment]
@rok-cesnovar - additional troubleshooting the install tips?

File: src/cmdstan-guide/installation.Rmd [hunk 1]
```diff
@@ -124,7 +124,7 @@ to the `PATH` environment variable.  If you have `Rtools40`, these should be:
 C:\RTools\RTools40\usr\bin
 C:\RTools\RTools40\mingw64\bin
 ```
-If you have and earlier version of RTools, e.g., RTools 3.6, use `RTools36` istead of `RTools40` in the above paths.
+If you have and earlier version of RTools, e.g., RTools 3.6, use `RTools36` instead of `RTools40` in the above paths.
```
Comment (rok-cesnovar) [inline src/cmdstan-guide/installation.Rmd:127]: Not the fault of this PR but this should be RTools35.

File: src/cmdstan-guide/installation.Rmd [hunk 2]
```diff
@@ -375,15 +375,66 @@ and in working order, run the following series of commands:
...
+‘mingw32-make’ is not recognised as an internal or external command
+```
+
+The solution is to call utility `pacman` as follows:
+
+```
+pacman -Sy mingw-w64-x86_64-make
+```
+
+See: https://discourse.mc-stan.org/t/cmdstan-installation-on-windows/11287/6
```
Comment (rok-cesnovar) [inline src/cmdstan-guide/installation.Rmd:423]: This is only true if you use the RTools 4.0 toolchain. 

Rtools35: 

- if mingw32-make, g++ or cut is not found, you need to add `C:\Rtools\mingw_64\bin` and `C:\Rtools\bin` to PATH

RTools40: 

- if mingw32-make, g++ or cut is not found you need to add  `C:\rtools40\mingw64\bin`  and `C:\rtools40\usr\bin`  and then run the pacman command. 

And that is it.

**Review by rok-cesnovar — APPROVED** [review]
Looks good to me. For Windows troubleshooting, I wrote a quick comment to expand a bit.

File: src/cmdstan-guide/installation.Rmd (hunk 2 above)
Comment (mitzimorris) [reply src/cmdstan-guide/installation.Rmd:423]: thanks!

**Pull request merged**